src/Uno.UI.Wasm/tsBindings/WindowManagerRegisterUIElementParams.ts

```diff
@@ -0,0 +1,61 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManagerRegisterUIElementParams
+{
+	/* Pack=4 */
+	TypeName : string;
+	IsFrameworkElement : boolean;
+	Classes_Length : number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'Classes_Length' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)